lib/puppet/pops/loader/loader.rb

```diff
@@ -29,7 +29,7 @@ class Loader
   attr_reader :loader_name
 
   # Describes the kinds of things that loaders can load
-  LOADABLE_KINDS = [:func_4x, :func_4xpp, :type_pp, :resource_type_pp, :plan, :task].freeze
+  LOADABLE_KINDS = [:func_4x, :func_4xpp, :datatype, :type_pp, :resource_type_pp, :plan, :task].freeze
```

Comment:
should it be `:type_rb` to be consistent with `:type_pp` ?

Reply:
Heh, I thought ':type_rb' would be inconsistent since no '_rb'  is used for functions :-)

Reply:
nevermind...